### PR TITLE
Salt Client registration from Command Line

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/13366

# Description

Initiat registration from Client (Salt)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/13366

### CODE REVIEW

**Comment by keichwa** [comment]
@paususe I guess this is what you asked for.

And @Loquacity do we want to have this backported to 4.1?

**Comment by Loquacity** [comment]
> And @Loquacity do we want to have this backported to 4.1?

Yes, let's, but wait until we unfreeze the repo (should be later this week).

File: modules/client-configuration/pages/registration-cli.adoc [hunk 1]
```diff
@@ -5,9 +5,10 @@
 // I'd say "no", according to the outcome of
 // https://github.com/SUSE/spacewalk/issues/9333 KE 2019-12-17
 
-Instead of the {webui}, you can use the command line to register a Salt client.
-This procedure requires that you have installed the Salt package on the Salt client before registration.
-For SLE 12 based clients, you also must have activated the [systemitem]``Advanced Systems Management`` module.
+In most cases, Salt clients are registered accurately with the default bootstrap methods.
+However, you can use Salt to register the client to the {productname} Server manually by editing the Salt minion file on the client, and providing the fully qualified domain name (FQDN) of the server.
+This method operates using ports 4505 and 4506 inbound to the server.
```
Comment (Loquacity) [inline modules/client-configuration/pages/registration-cli.adoc:10]:
```suggestion
This method uses ports 4505 and 4506 inbound to the server.
```

File: modules/client-configuration/pages/registration-cli.adoc [hunk 2]
```diff
@@ -17,11 +18,14 @@ Use the bootstrap script procedure to register traditional clients.
 For more information, see xref:registration-bootstrap.adoc[].
 ====
 
+This procedure requires that you have installed the Salt package on the Salt client before registration ([package]``salt-minion``).
```
Comment (Loquacity) [inline modules/client-configuration/pages/registration-cli.adoc:21]:
```suggestion
This procedure requires that you have installed the [package]``salt-minion`` package on the Salt client before registration.
```

File: modules/client-configuration/pages/registration-cli.adoc [hunk 3]
```diff
@@ -32,13 +36,11 @@ or:
 ----
 /etc/salt/minion.d/NAME.conf
 ----
-+
-This is sometimes also called a minion file.
-. Add the {productname} Server or Proxy FQDN as the `master`, and the activation key, to the client configuration file:
+. Add or edit the FQDN of the {productname} Server or of the Proxy as the `master`, and the activation key, to the client configuration file:
```
Comment (Loquacity) [inline modules/client-configuration/pages/registration-cli.adoc:39]:
```suggestion
. Add or edit the FQDN of the {productname} Server or of the Proxy as the `master`, and add the activation key to the client configuration file:
```

??

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by Loquacity — APPROVED** [review]

**Comment by keichwa** [comment]
And then I'll create a 4.1 backport.

**Pull request merged**